studio/components/interfaces/App/RouteValidationWrapper.tsx

```diff
@@ -16,6 +16,9 @@ const RouteValidationWrapper = ({ children }: PropsWithChildren<{}>) => {
   const { ref, slug } = useParams()
   const navLayoutV2 = useFlag('navigationLayoutV2')
 
+  const { session } = useAuth()
+  const isLoggedIn = Boolean(session)
```

Comment:
You can use `useIsLoggedIn()` from `lib/auth` here :)

Reply:
oooh okay gotcha - i'll update accordingly 🙏

Reply:
just pushed :)